src/sass/components/_signup.scss

```
@@ -0,0 +1,33 @@
+.input-item {
+  @include flex(column, flex-start, flex-start);
+  gap: 20px;
+
+  span {
+    font-size: 14px;
+    font-weight: 500;
+
+    b {
+      color: $red;
+      margin-left: 5px;
+    }
+  }
+
+  & > input {
+    width: 100%;
+    height: 50px;
+    font-size: 14px;
+    padding: 0 20px;
+    border: 1px solid $line-gray;
+  }
+}
+
+.item-checkbox {
```

```suggestion
.checkbox {
```
Này tách ra 1 file riêng luôn. Vì nó là 1 phạm trù/ 1 component riêng